src/automations/defaultActionValues.ts

```diff
@@ -58,6 +58,24 @@ Flow run URL: {{ flow_run|ui_url }}
 State message: {{ flow_run.state.message }}
 `.trim()
 
+const NOTIFICATION_BODY_CUSTOM = `
+Automation: {{ automation.name }}
+Description: {{ automation.description }}
+
+Event: {{ event.id }}
+Resource:
+{% for label, value in event.resource %}
+{{ label }}: {{ value }}
+{% endfor %}
+Related Resources:
+{% for related in event.related %}
+    Role: {{ related.role }}
+    {% for label, value in related %}
+    {{ label }}: {{ value }}
+    {% endfor %}
+{% endfor %}
```

Comment:
This is from the bottom of the [automation docs](https://docs.prefect.io/latest/concepts/automations/?h=automation#templating-with-jinja)

> In addition to those shortcuts for flows, deployments, and work pools, you have access to the automation and the event that triggered the automation. See the [Automations API](https://app.prefect.cloud/api/docs#tag/Automations) for additional details.